client/post-editor/editor-revisions-list/index.jsx

```diff
@@ -113,9 +118,9 @@ class EditorRevisionsList extends PureComponent {
 			comparisons,
 			postId,
 			revisions,
+			isRequestingRevisions,
 			selectedRevisionId,
 			siteId,
-			newRevisionIds,
 		} = this.props;
 		const classes = classNames( 'editor-revisions-list', {
 			'is-loading': isEmpty( revisions ),
```

Comment:
We should probably add `|| isRequestingRevisions` here, to add the loading effect to the revision count.